contracts/sol/MACI.sol

```diff
@@ -71,7 +71,11 @@ contract MACI is DomainObjs, ComputeRoot, MACIParameters, VerifyTally {
     // immediately upon deployment.
     uint256 public signUpTimestamp;
 
-    // Duration of the sign-up and voting periods, in seconds
+    // Duration of the sign-up and voting periods, in seconds. If these values
+    // are set to 0, the contract will be in debug mode - that is, only the
+    // coordinator may sign up and publish messages. This makes it possible to
+    // submit a large number of signups and messages without having to do so
+    // before the signup and voting deadlines.
```

Comment:
Is debug mode supposed to be used in production?

Reply:
No, since in production the signup and voting periods will be greater than 0, debug mode will be off.

Reply:
If it is meant to be used solely during development, could it be implemented in a separate contract (e.g. `MACIDebug.sol`) to prevent MACI contract from increasing in size?

Reply:
(The MACI contract itself is not that big yet, but in clrfund there's a MACI factory contract which size is close to EIP170 limit.)